src/entities/BaseEntity.java

```diff
@@ -0,0 +1,67 @@
+package entities;
+
+import java.util.Date;
+
+public abstract class BaseEntity<T> {
+    private Class<T> entity;
+    private Long updatedby;
+    private Long createdBy;
+    private String status;
+    private Date createdAt;
```

Comment:
What do you think about usong ZonedDatetime instead?

Reply:
No problem, I've researched that java.util.Date is a bit obsolete and represents a moment in UTC, which isn't necessary, and ZoneDateTime is used more instead

Reply:
Nice, you can resolve this comment once you update the pr

Reply:
solved